### ISSUE TITLE
Server Crash when encountering snow biome

### ISSUE THREAD

**Issue by Tomma1** [LunaPixelStudios/Better-MC#1047]
### Describe the Bug

While exploring the world on our Server (Bisect Hosting), we found a modded snowy area where the server would crash. At first, I managed to solve this problem by preloading the chunks, but now we are stuck there. When we load into the server, it freezes preventing me from doing anything. 

I installed the modpack on the server via the pre install service Bisect Hosting offers. 

### Reproduction Steps

1. Walk around the server
2. Encounter Snowy biome
3. Server Freezes

### Crash Report or Log

https://gist.github.com/Tomma1/01bb2b1c0e0e80e0db1512b62b8e0fcd

### Operating System

Windows 10

### Minecraft Version

1.19

### Modloader

Forge

### Modpack Version

bmc3 v12

### Optifine

No

### Other Mods

Chunk-Pregenerator
World Edit (Client side)

**Comment by SHXRKIIE** [comment]
Can you see if removing Chunk-Pregenerator and smooth chunk mod?

**Comment by Tomma1** [comment]
> Can you see if removing Chunk-Pregenerator and smooth chunk mod?

It didnt change anything. The server still freezes when entering that region. 
As for an update, I ended up having to modify our .dat positions to get us out of there. Since this problem first occurred, the world has been acting strange, things from people's inventory sometimes disappear. Gradually, not instantly (I did not edit anything else in the dat files). According to one player, new caves appeared also. I'm not sure if this is part of some auto-updating feature, or if something is really wrong. I will attach the log (since it didn't generate any crash reports) about what I found weird. You will probably be able to see more. (its the latest log)
https://gist.github.com/Tomma1/5153f7aefb41faa445543de10a44d905
